audio.py

```diff
@@ -30,9 +35,32 @@ def read_audio(
 		elif audio_path.endswith('.wav'):
 			sample_rate_, signal = scipy.io.wavfile.read(audio_path)
 			signal = torch.as_tensor(signal[None, :] if len(signal.shape) == 1 else signal.T)
+		elif backend == 'soundfile' and audio_path.endswith('.gsm'):
```

Comment:
Please do a backend == 'soundfile' check instead (https://github.com/vadimkantorov/convasr/pull/14#discussion_r461875737). Why restricting `soundfile` backend for `gsm` files only

Reply:
done